### PR TITLE
feat(grafana): Install and configure grafana plugin

### LINKED ISSUE #15: Plugin: Install and configure Grafana plugin
Provide a connection to Operate First grafana instance enabling Backstage to link all the dashboards from individual components.

https://github.com/K-Phoen/backstage-plugin-grafana/

### PR DESCRIPTION
**Pull request by SamoKopecky**
Resolves #15 
- Install and configure grafana plugin
- Update `README.md` with grafana plugin section
- Add  `GRAFANA_TOKEN` variable

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "node": "14 || 16"
   },
   "scripts": {
-    "dev": "BACKEND_SECRET=secret concurrently \"yarn start\" \"yarn start-backend\"",
+    "dev": "set -a && source ./.env && set +a && BACKEND_SECRET=secret concurrently \"yarn start\" \"yarn start-backend\"",
```
Comment (tumido) [inline package.json:9]: Hm, can you please move the `BACKEND_SECRET` to the `.env` file as well?

I don't like the messy set+source+set, it's a shame backstage cli doesn't support dotenv... Can you please instead try installing `env-cmd` as a dev dependency to root of all workspaces and extend the `start-backend` script with it? We shouldn't need env variables on the frontend. https://github.com/toddbluhm/env-cmd

**Review by tumido — CHANGES_REQUESTED** [review]
I think README typos and repo pointers should be in a separate PR, it's unrelated to Grafana. Maybe the conversion of env variables to `.env` file should be a separate PR as well.. Let's keep PRs atomic if possible.

**Comment by tumido** [comment]
> Maybe the conversion of env variables to `.env` file should be a separate PR as well.. Let's keep PRs atomic if possible.

https://github.com/operate-first/service-catalog/pull/34 is precisely why that should be a separate PR. PR #34  is dependent on grafana plugin now, which shouldn't be the case.

File: README.md [hunk 2]
```diff
@@ -10,21 +10,21 @@ The service catalog is implemented using [backstage](https://backstage.io/). Thi
 
 Backstage can utilize a separate location (for example a GitHub repository) for the catalog component definitions. More information on how these files are defined can be found in the backstage [documentation](https://backstage.io/docs/features/software-catalog/descriptor-format).
 
-Component definition files for this service-catalog are located [here](https://github.com/SamoKopecky/apps/tree/master/service-catalog). If you want to edit them, make a pull request in that repository.
+Component definition files for this service-catalog are located [here](https://github.com/operate-first/apps/tree/master/service-catalog). If you want to edit them, make a pull request in that repository.
 
-There is an `example` folder located that only contains catalog items dedicated to local development. Switching between local and remote sources can configured in `app-config.local.yaml` in the `catalog.locations` key.
+There is an `example` folder located that only contains catalog items dedicated to local development. Switching between local and remote sources can be configured in `app-config.local.yaml` in the `catalog.locations` key.
```
[the remark this answers is not in the document]
Comment (SamoKopecky) [reply README.md:15]: Moved to another PR.

File: README.md [hunk 3]
```diff
@@ -10,21 +10,21 @@ The service catalog is implemented using [backstage](https://backstage.io/). Thi
...
 
 ## Permissions
 
 Right now the access to the catalog is read-only for everyone. Permissions are defined using a [permission policy](https://backstage.io/docs/permissions/writing-a-policy). A permission policy is defined in `packages/backend/src/plugins/permission.ts`.
 
 ## Integrations
 
-It is possible to integrate backstage using many other external providers such as GitHub or GitLab. For now we decided that using no integrations is the best option. Read more about [integrations](https://backstage.io/docs/integrations/).
+It is possible to integrate backstage using many other external providers such as GitHub or GitLab. For now, we decided that using no integrations is the best option. Read more about [integrations](https://backstage.io/docs/integrations/).
```
[the remark this answers is not in the document]
Comment (SamoKopecky) [reply README.md:23]: Moved to another PR.

File: README.md [hunk 4]
```diff
@@ -10,21 +10,21 @@ The service catalog is implemented using [backstage](https://backstage.io/). Thi
...
 
 ## Configuration files
 
-There are 4 configuration files present in this repository. Backstage handles configuration files similarly to how `kustomize` handles deployment manifests. There is a base configuration file, in this case `app-config.yaml`. Then 3 other configuration files add on top of the base config file by merging the two files.
+There are 4 configuration files present in this repository. Backstage handles configuration files similarly to how `kustomize` handles deployment manifests. There is a base configuration file, in this case, `app-config.yaml`. Then 3 other configuration files add on top of the base config file by merging the two files.
```
[the remark this answers is not in the document]
Comment (SamoKopecky) [reply README.md:27]: Moved to another PR.

File: app-config.local.yaml [hunk 5]
```diff
@@ -26,4 +26,4 @@ catalog:
       target: ../../examples/all.yaml
     #  Uncomment to use defintions files from apps repo
     # - type: url
-    #   target: https://github.com/SamoKopecky/apps/blob/master/service-catalog/all.yaml
+    #   target: https://github.com/operate-first/apps/blob/master/service-catalog/all.yaml
```
[the remark this answers is not in the document]
Comment (SamoKopecky) [reply app-config.local.yaml]: Moved to another PR.

File: package.json (hunk 1 above)
Comment (SamoKopecky) [reply package.json]: Moved to another PR.

**Comment by SamoKopecky** [comment]
Waiting for #41 to merge so I can add a variable to `.env.example` file.
/hold

**Review by tumido — APPROVED** [review]
/lgtm
/approve

ready to merge once rebased

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**